deployment/node-feature-discovery/values.yaml

```diff
@@ -1,5 +1,6 @@
 image:
   repository: gcr.io/k8s-staging-nfd/node-feature-discovery
+  version: master
```

Comment:
Currently `appVersion` is set to `master` in `Chart.yaml`
When a chart is released `appVersion` will be updated with nfd release tag so the correct image is pulled.
with this change, this field would need to be updated as well.

IMO this should be empty by default and used as image tag only if specified by the user. (i.e image tag override.)

Reply:
But what if an user/developer wants to use HELM to deploy a devel version of NFD, how can I override the TAG value? is there a way we can set this from the command line? `helm install .... --set image.tag=devel` ?

Reply:
yes exactly. 

a user may override. but if not specified, use `.Chart.AppVersion`

Reply:
Are you happy with this? Can we resolve the conversation?

Reply:
yes